### ISSUE TITLE
addSubvariable(s) doesn't retain description

### ISSUE THREAD

**Issue by persephonet** [Crunch-io/rcrunch#200]
`[crunch] > description(ds$allpets)
[1] "Do you have any of these animals as pets? Please select all that apply."
[crunch] > ds$allpets <- addSubvariables(ds$allpets, VariableDefinition(data=factor('selected', levels=c('selected', 'not selected')), alias='testing_add_sub', name='testing_add_sub'))
[crunch] > description(ds$allpets)
[1] ""
[crunch] > description(ds$allpets) <- 'Do you have any of these animals as pets? Please select all that apply.'
[crunch] > ds$allpets <- addSubvariable(ds$allpets, VariableDefinition(data=factor('selected', levels=c('selected', 'not selected')), alias='testing_add_sub', name='testing_add_sub'))
[crunch] > description(ds$allpets)
[1] ""
[crunch] > self(ds)
[1] "https://app.crunch.io/api/datasets/f79b4b0cd1094b2aa050cfd079e18bf9/"`

**Comment by nealrichardson** [comment]
Thanks Persephone. Looks like something on the server inside of [this PATCH request](https://github.com/Crunch-io/rcrunch/blob/master/R/add-subvariable.R#L23-L24) is dropping metadata. @jonkeane could you please write up a ticket for that?

**Comment by jonkeane** [comment]
Absolutely, I've written it up as Pivotal issue #155014863, with a note to resolve this issue when it's addressed.
